src/OpcApplicationConfiguration.cs

```diff
@@ -75,117 +77,117 @@ public OpcApplicationConfiguration()
+                .SetMaxPublishRequestCount(MAX_PUBLISH_REQUEST_COUNT)
+                .SetMaxRequestThreadCount(MAX_REQUEST_THREAD_COUNT)
+            // LDS registration interval
+                .SetMaxRegistrationInterval(LdsRegistrationInterval);
 
             // security configuration
-            await InitApplicationSecurityAsync().ConfigureAwait(false);
+            ApplicationConfiguration = await InitApplicationSecurityAsync(securityBuilder).ConfigureAwait(false);
 
-            // set LDS registration interval
-            ApplicationConfiguration.ServerConfiguration.MaxRegistrationInterval = LdsRegistrationInterval;
-            Logger.Information($"LDS(-ME) registration interval set to {LdsRegistrationInterval} ms (0 means no registration)");
+            foreach (var policy in ApplicationConfiguration.ServerConfiguration.SecurityPolicies)
+            {
+                Logger.Information($"Security policy {policy.SecurityPolicyUri} with mode {policy.SecurityMode} added");
+                if (policy.SecurityMode == MessageSecurityMode.None)
+                {
+                    Logger.Warning("Note: This is a security risk and needs to be disabled for production use");
+                }
+            }
 
-            // show certificate store information
-            await ShowCertificateStoreInformationAsync().ConfigureAwait(false);
+            Logger.Information($"LDS(-ME) registration intervall set to {LdsRegistrationInterval} ms (0 means no registration)");
 
-            // Support larger number of nodes.
-            ApplicationConfiguration.ServerConfiguration.MaxMessageQueueSize = MAX_MESSAGE_QUEUE_SIZE;
-            ApplicationConfiguration.ServerConfiguration.MaxNotificationsPerPublish = MAX_NOTIFICATIONS_PER_PUBLISH;
-            ApplicationConfiguration.ServerConfiguration.MaxSubscriptionCount = MAX_SUBSCRIPTION_COUNT;
-            ApplicationConfiguration.ServerConfiguration.MaxPublishRequestCount = MAX_PUBLISH_REQUEST_COUNT;
-            ApplicationConfiguration.ServerConfiguration.MaxRequestThreadCount = MAX_REQUEST_THREAD_COUNT;
+            // configure OPC stack tracing
+            Utils.SetTraceMask(OpcStackTraceMask);
+            Utils.Tracing.TraceEventHandler += LoggerOpcUaTraceHandler;
+            Logger.Information($"opcstacktracemask set to: 0x{OpcStackTraceMask:X}");
 
-            return ApplicationConfiguration;
-        }
+            // log certificate status
+            var certificate = ApplicationConfiguration.SecurityConfiguration.ApplicationCertificate.Certificate;
+            if (certificate == null)
+            {
+                Logger.Information($"No existing Application certificate found. Create a self-signed Application certificate valid from yesterday for {CertificateFactory.DefaultLifeTime} months,");
+                Logger.Information($"with a {CertificateFactory.DefaultKeySize} bit key and {CertificateFactory.DefaultHashSize} bit hash.");

```

Comment:
This will generate two log lines with two different timestamps. Pls merge the string instead.